index.md

```diff
@@ -18,6 +18,37 @@ Doxygen documentation is available:
 * for [v1.6](./v1.6/doxygen/index.html)
 * for [v1.5](./v1.5/doxygen/index.html)
 
+#### Releases' support status
+
+Currently last 3 branches/releases are fully supported. Latest releases can be
+seen on the ["releases" tab on the GitHub page](https://github.com/pmem/libpmemobj-cpp/releases).
+
+| Version branch | First release date | Maintenance status |
+| -------------- | ------------------ | ------------------ |
+| stable-1.9 | Jan 31, 2020 | Full |
+| stable-1.8 | Oct 3, 2019 | Full |
+| stable-1.7 | Jun 26, 2019 | Full |
+| stable-1.6 | Mar 15, 2019 | Limited |
+| stable-1.5 | Oct 26, 2018 | Limited |
```

Comment:
I think listing dates and version numbers of last releases from each branch is useful in determining how serious we are about this (and I know libpmemobj-cpp is lagging on this front).